prism-clients-generator/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+plugins {
+    kotlin("jvm")
+    `maven-publish`
+}
+
+group = "io.iohk.atala"
+description = "Prism gRPC clients generator. Generate convenient definitions for JS/JVM/iOS/Android targets"
```

Comment:
This is not a PRISM client generator anymore but a gRPC client generator, I'd try to keep this generic with the goal to get it merged to the upstream project (https://github.com/streem/pbandk).

Reply:
@AlexITC I am totally agree, and I called the project `grpc-clients-generator` at first but there is a tiny issue why I decided to use `prism` in the name and everywhere, it's `GrpcClient` class. I mean those generated clients are relying on the `GrpcClient` which has methods with `PrismMetadata`.

Also all generated clients have methods which accept `PrismMetadata`, so I concluded that for now the generator is tied with `prism-protos` `GrpcClient`.  Should we do something about this?
Also, those auth and not auth methods seem to be prism-specific.

For now I would leave as it's and later think if we can make it general to merge to upstream. what do you think?

Reply:
yeah, I agree with Ilya -- Dan started two discussions about potentially useful features for upstream: https://github.com/streem/pbandk/discussions/173 & https://github.com/streem/pbandk/discussions/174

but this one looks more iohk-specific and it's not obvious how to make it useful for someone else

Reply:
@pva701 thanks for the explanation, I'd do this:
- Get this merged to unblock the work.
- Create a task for refactoring this generator in a way that it isn't prism-specific but generic enough to allow including the prism-specific functionality when depending on the generator.
- Get the dependent PR merged too.